### ISSUE TITLE
Attempt to change capitalization of alternate name results in an error

### ISSUE THREAD

**Issue by FreddieChopin** [theCrag/website#3706]
**What happened?**

1. Take any test route (possibly the same issue for areas, did not check), for example this one - https://www.thecrag.com/climbing/test-area/route/528006219
2. Add any alternate name, for example "xxx"
3. Save
4. Edit the route again
5. Try to change the capitalization of the alternate name, for example from "xxx" to "Xxx"
6. The system shows an error `{ "alternateNamesError" : "name 'Xxx' already exists, so cannot be created." }`

Ideally also provide screenshots:
![Screenshot from 2020-06-17 19-53-56](https://user-images.githubusercontent.com/1139038/84932277-5abb4880-b0d4-11ea-9921-b7cb6ec5fd94.png)

The only option to change the capitalization is to do it in several steps - first change it to something different (say "xxxyyy") then change to what you really want (in this example "Xxx").

**What you expected:**

* [ ] Changing just the capitalization of alternate names should be possible just the same as it is possible for the "main" name

Possibly related to #2974 and #3005

**Comment by Mdemaillard** [comment]
Is this working for the main name ? (recapitalizing in 1 step?) see #2547 as well for context - protection about incorrect renaming is too aggressive

**Comment by FreddieChopin** [comment]
> Is this working for the main name ?

Yes, no problem there. That's why I think it should be the same for alternate names.
